CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ This project tries to follow [Semantic Versioning](http://semver.org/) since the
 
 This document mainly describes API changes important to users of this library.
 
+## 1.13.0 - 2020-09-25
```

Comment:
Does that need a minor version bump? Wouldn't patch be enough 

(in the end, doesn't matter much, they're free anyway ;))

Reply:
yeah .. good point.
especially since it is not a version bump but just an expansion.

will turn it into a 1.12.1

Reply:
done